### PR TITLE
Fasih/UPM-642/Enabled POA for tablet view

### PR DESCRIPTION
**Pull request by fasihali-deriv**
## Changes:

Enabled POA for tablet view

### Screenshots:

Screenshot 2024-05-21 at 3 02 53 PM
Please provide some screenshots of the change.

### CODE REVIEW

File: packages/account/src/Components/scrollbars-container/scrollbars-container.tsx [hunk 1]
```diff
@@ -6,19 +6,27 @@ import { isMobile } from '@deriv/shared';
...
-    <ThemedScrollbars is_bypassed={isMobile()} height={scroll_offset ? `calc(100% - ${scroll_offset})` : '100%'}>
-        <div
-            className={classNames('account__scrollbars_container', className)}
-            data-testid='dt_scrollbar_container_div'
+    isFullHeight = false,
+}: React.PropsWithChildren<TScrollbarsContainer>) => {
+    const height_unit = isFullHeight ? '100vh' : '100%';
+    return (
+        <ThemedScrollbars
+            is_bypassed={isMobile()}
```
Comment (utkarsha-deriv) [inline packages/account/src/Components/scrollbars-container/scrollbars-container.tsx]: Is the themed scrollbar bypassed only for mobile? shall we make it !isDesktop 🙏

File: packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.tsx [hunk 2]
```diff
@@ -271,7 +272,7 @@ const ProofOfAddressForm = observer(
         }
         const setOffset = (status: { msg: string }) => {
             const mobile_scroll_offset = status?.msg ? '200px' : '154px';
-            return is_mobile && !is_for_cfd_modal ? mobile_scroll_offset : '80px';
+            return (isMobile || isTablet) && !is_for_cfd_modal ? mobile_scroll_offset : '80px';
```
Comment (utkarsha-deriv) [inline packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.tsx]: just curious, why not !isDesktop 😄 ?

File: packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.tsx [hunk 3]
```diff
@@ -283,21 +284,21 @@ const ProofOfAddressForm = observer(
...
                                 >
-                                    <FormBody scroll_offset={setOffset(status)}>
+                                    <FormBody scroll_offset={setOffset(status)} isFullHeight={!isDesktop}>
                                         {(status?.msg || is_resubmit) && (
                                             <HintBox
                                                 className='account-form_poa-submit-error'
                                                 icon='IcAlertDanger'
                                                 message={
-                                                    <Text as='p' size={is_mobile ? 'xxxs' : 'xs'}>
+                                                    <Text as='p' size={isMobile ? 'xxxs' : 'xs'}>
```
Comment (utkarsha-deriv) [inline packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.tsx]: this text size just for mobile or tablet view also?

File: packages/account/src/Components/scrollbars-container/scrollbars-container.tsx (hunk 1 above)
Comment (fasihali-deriv) [reply packages/account/src/Components/scrollbars-container/scrollbars-container.tsx]: True let me update this. 
BTW i did not add this prop it was already there 😁 

Thanks @utkarsha-deriv for highlighting

File: packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.tsx (hunk 2 above)
Comment (fasihali-deriv) [reply packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.tsx]: Ohh yes. let me update this as well. 😬 
Thanks @utkarsha-deriv for highlighting

File: packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.tsx (hunk 3 above)
Comment (fasihali-deriv) [reply packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.tsx]: huhh. Actually I forgot this as well 

thanks @utkarsha-deriv 🙇

File: packages/account/src/Components/poa/common-mistakes/common-mistake-examples.scss [hunk 4]
```diff
@@ -12,6 +12,9 @@
             column-gap: calc((100% - 23.7rem * 3) / 2);
             row-gap: 3.2rem;
         }
+        @include tablet-screen {
```
Comment (amina-deriv) [inline packages/account/src/Components/poa/common-mistakes/common-mistake-examples.scss:15]: clarification: how is this handled in mobile?

Comment (fasihali-deriv) [reply packages/account/src/Components/poa/common-mistakes/common-mistake-examples.scss:15]: **This is tablet view**

Screenshot 2024-05-21 at **3** 47 13 PM

**This is Mobile view**

This mobile view this same as we have in production.


Screenshot 2024-05-21 at 3 48 18 PM

File: packages/account/src/Components/scrollbars-container/scrollbars-container.tsx [hunk 5]
```diff
@@ -1,24 +1,33 @@
...
-    <ThemedScrollbars is_bypassed={isMobile()} height={scroll_offset ? `calc(100% - ${scroll_offset})` : '100%'}>
-        <div
-            className={classNames('account__scrollbars_container', className)}
-            data-testid='dt_scrollbar_container_div'
+    isFullHeight = false,
+}: React.PropsWithChildren<TScrollbarsContainer>) => {
+    const { isDesktop } = useDevice();
+    const height_unit = isFullHeight ? '100vh' : '100%';
+    return (
+        <ThemedScrollbars
```
Comment (likhith-deriv) [inline packages/account/src/Components/scrollbars-container/scrollbars-container.tsx:21]: This change will impact all components using <FormBody> 👀 . Please ensure they are tested

Comment (fasihali-deriv) [reply packages/account/src/Components/scrollbars-container/scrollbars-container.tsx:21]: Hi @likhith-deriv your concern is valid but as I made default value false so it should not break anything 😬 
But still I will check other screens as well. 

Reason why I added this. 

Without adding this 100vh I am not able to scroll till down properly on mobile and tablet screen.

File: packages/account/src/Components/scrollbars-container/scrollbars-container.tsx (hunk 1 above)
Comment (utkarsha-deriv) [reply packages/account/src/Components/scrollbars-container/scrollbars-container.tsx]: Thank you 🙏

**Review by utkarsha-deriv — APPROVED** [review]

**Review by amina-deriv — APPROVED** [review]

**Review by likhith-deriv — APPROVED** [review]

**Pull request merged**